src/style.css

```
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bar--1 {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 24px;
+  background-size: 19px;
+  height: 70px;
+}
+
+.bar--2 {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 12px 14px;
+  background-size: 11px;
+  height: 42px;
+}
+
+.bar {
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  box-shadow: 0 1 8px 0 10% #3d4e61;
+  border-radius: 4px;
+  background-color: #fff;
+  box-sizing: border-box;
+}
+
+.input--big {
+  margin-top: 25px;
+  margin-left: 62px;
+  margin-bottom: 23px;
+  border: 0;
+  font-size: 16px;
+  color: #3d4e61;
+  width: 129px;
+  height: 22px;
+  line-height: 21.86px;
+}
+
+.input--small {
+  margin-top: 12px;
+  margin-bottom: 10px;
+  margin-left: 33px;
+  margin-right: 0;
+  border: 0;
+  font-size: 14px;
+  font-weight: 400;
+  width: 167.29px;
+  height: 20px;
+  line-height: 19.12px;
+}
+
+.input {
+  color: #3d4e61;
+}
+
+.bar:hover {
+  box-shadow: 0 3 8px 0 20% #3d4e61;
```

let's use same value as in mock design 
```suggestion
  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
```